schemas/blocks/relationship_collector.json

```diff
@@ -26,6 +27,9 @@
       "question_variants": {
         "$ref": "definitions.json#/question_variants"
       },
+      "unrelated_relationship_block": {
```

Comment:
I think `unrelated_block` is better. I know there was some concern over it being mis-interpreted as a block that has nothing to do with the parent block, but if that was the case it wouldn't be here 🙂 .